src/katgpucbf/utils.py

```diff
@@ -20,14 +20,19 @@
 import logging
 import signal
 from asyncio import get_event_loop
+from enum import Enum
+from typing import Optional, TypeVar
```

Comment:
Rather remove Optional and use `T | None` for consistency with the rest of the katgpucbf codebase.